pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
     "ipywidgets",
     "pydantic>=2.0",
     "requests",
-    "qiskit>=1.0, <1.2",
+    "qiskit==1.2.4",
```

Comment:
~If we can, it's usually better to not pin versions explicitly when releasing a package (otherwise, in this case it won't allow minor updates, and it might prevent the user from installing qiskit-alice-bob-provider alongside other packages requiring a different qiskit version). Do we have a good reason to believe our package should only be installed with this specific version of qiskit ? Or can we keep something like `"qiskit>=1.2, <1.3"` ?~

edit : just saw the resolved conversation with @sbrieuc, I'm fine with this then ! (just a small comment or commit message to explain it would be great for next time, so that we can easily see why this version was pinned when we'll need to upgrade it again)

Reply:
I added a comment